staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/oidc/oidc.go

```diff
@@ -288,7 +288,7 @@ func New(opts Options) (*Authenticator, error) {
 		// in the authentication configuration.
 		// The go oidc library does not support validating
 		// multiple audiences, so we have to skip the client ID check and do it ourselves.
-		// xref: https://github.com/coreos/go-oidc/issues/397
+		// xref: https://github.com/coreos/go-oidc/v3/oidc/issues/397
```

Comment:
heh, revert this bit, the old link was correct

Reply:
find and replace issue :) fixed now!